Services/CO.CDP.Organisation.WebApi/UseCase/LookupOrganisationUseCase.cs

```diff
@@ -3,11 +3,58 @@
 using CO.CDP.OrganisationInformation.Persistence;
 
 namespace CO.CDP.Organisation.WebApi.UseCase;
-public class LookupOrganisationUseCase(IOrganisationRepository organisationRepository, IMapper mapper) : IUseCase<string, Model.Organisation?>
+public class LookupOrganisationUseCase(IOrganisationRepository organisationRepository, IMapper mapper) : IUseCase<OrganisationQuery, Model.Organisation?>
```

Comment:
Service registration in `Services/CO.CDP.Organisation.WebApi/Program.cs` will need to be updated too.

Reply:
Hi Kuba, Updated this.